lib/config.js

```diff
@@ -49,8 +50,8 @@ class Pattern {
 }
 
 class UrlPattern extends Pattern {
-  constructor (url, type) {
-    super(url, false, true, false, false, type)
+  constructor (url, type, integrity) {
+    super(url, false, true, false, false, type, false, integrity)
```

Comment:
nit: Should it be `undefined` instead of `false` for the `isBinary`?

Reply:
I set false instead of undefined because all the other parameters are passed boolean as the default value. Should only the isBinary be passed undefined?

Reply:
In practice is is [never used](https://github.com/karma-runner/karma/blob/5e71cf591e9006ad2621dd88d09739bfbffd463c/lib/file-list.js#L66) for the absolute URL patterns, but IMO it is better to set it to `undefined` because it used to be `undefined` before this PR and also to avoid confusion for the future reader.

Having said that, I don't think it should block this PR.

Reply:
OK, I'll fix it.

Reply:
done.